### PR TITLE
Chat: ensure ScrollDown button only takes it's width

### PR DESCRIPTION
**Pull request by ichim-david**
Fixes https://github.com/sourcegraph/cody/issues/6133

This pull request allow interaction with interactive elements if we can see them. 
Previously the button parent took full width making it impossible to interact with the other elements unless scrolling.

## Test plan
Scroll until you see the Scroll to End button.
Check that you can interact with the buttons such as Copy or Execute.

Here are some screenshots with Cody within VSCode  and with the storybook for ScrollDown
storybook-text-select-not-possible
storybook-text-select
cody-scroll-to-end-broken
cody-scroll-to-end-fixed

## Changelog
Chat: ensure ScrollDown button only takes it's width allowing the interaction with other elements

### CODE REVIEW

**Comment by ichim-david** [comment]
@vovakulikov I noticed you worked with ScrollDown component in https://github.com/sourcegraph/cody/pull/5386.
Would love to get a review from you or maybe you can CC someone else if that person is better suited to review this work.
CC @jdorfman and thank you guys in advance.

**Comment by vovakulikov** [comment]
@ichim-david sure, styles changes over scroll down button look fine, this PR has some other unrelated as I see changes, let me know when it's ready for official review.

**Review by vovakulikov — APPROVED** [review]

**Comment by ichim-david** [comment]
@beyang what's the review and merging workflow for community contributions like this one?

I got approval from @vovakulikov but it's not clear to me who does the merging, if someone from 
a release team needs to think about when it should be merged, if for such a simple fix all of the tests
needs to run and if so tests should be allowed to run for Contributor roles.

TLDR I want to follow the right process as long as I know what this process is like :)

**Comment by beyang** [comment]
@ichim-david we're currently in the midst of revamping how we review community contributions. For now, just tag @jdorfman and he'll surface them me or the appropriate eng. This obviously doesn't scale, but it's a stopgap while we pin down a process that works with our internal eng cadence. Apologies for the delays on reviews—thank you so much for these contributions and for bearing with us on the current process pain points :pray:
